### DISCUSSION TITLE
Getting the examples to work on a LyraT 4.3 board

### DISCUSSION THREAD

**Discussion by technoblogy** [pschatzmann/arduino-audio-tools#718] [Q&A] [answered]
I have both an AI Thinker AudioKit board and an ESP32 LyraT 4.3 board.

I’ve successfully got the AI Thinker AudioKit board playing Internet radio streams through the on-board amplifiers to speakers, using your example file:

    player-url_icy-audiokit

I’ve now tried the LyraT board. I wasn’t sure what example file to use, so tried:

    player-url_icy-i2s

It connects to the stream correctly and I get as far as a series of lines:

    [I] AudioCopy.h : 136 - StreamCopy::copy 1024 -> 1024 -> 1024 bytes - in 1 hops
    [I] AudioCopy.h : 136 - StreamCopy::copy 1024 -> 1024 -> 1024 bytes - in 1 hops

but no sound. Is there something else I need to do to enable the amplifier, and the touch buttons?

Sorry if this is already in the documentation somewhere, and thank you for an amazing resource!

**Comment by pschatzmann** [comment, marked as answer]
Check the Readme of the AudioKit project! 
You need to select the proper board

All these boards need the AudioKitStream!

**Comment by technoblogy** [comment]
OK, thank you - I'll try that.

**Comment by technoblogy** [comment]
I've now selected the LyraT 4.3 board in AudioKitSettings.h with:

    #define AUDIOKIT_BOARD 1

I can compile and upload:

basic-a2dp-audiokit
player-sd-audiokit
player-url_icy-audiokit

but I can't get any sound through the speakers attached to the JST sockets. Please can someone suggest what I'm doing wrong.

The same tests with the AI Thinker AudioKit board worked perfectly first time.
